src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -297,6 +303,19 @@ void DlgPrefInterface::slotSetScheme(int) {
     skinPreviewLabel->setPixmap(m_pSkinLoader->getSkinPreview(m_skin, m_colorScheme));
 }
 
+void DlgPrefInterface::slotSetSkinDescription(QString skin) {
+    SkinManifest manifest = LegacySkinParser::getSkinManifest(
+            LegacySkinParser::openSkin(m_pSkinLoader->getSkinPath(skin)));
+    QString description;
+    if (manifest && manifest.has_description()) {
```

Comment:
I added `manifest` to test if it's not Null but here it doesn't work that way, compiler complains.
How can I test properly? `manifest.IsInitialized()`

Reply:
Is that test necessary at all?

Reply:
I don't think so. It's not a pointer.

Reply:
yeah, wedon't need it. `LegacySkinParser::openSkin` would thrown an error and `manifest.has_description()` just returns false.
will revert